.gitignore

```diff
@@ -4,4 +4,5 @@ node_modules
 .DS_Store
 vendor/*
 mojblocks.zip
+build/*
```

Comment:
This wasalso added by adam and is in master.. not sure why the compare isnt working.